### PR TITLE
Prow auto bumper won't bother anyone but oncall

### PR DESCRIPTION
**Pull request by coryrc**
No reason to auto-add reviewers; it's the oncall's responsibility to review.

### CODE REVIEW

File: tools/prow-auto-bumper/pullrequest.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func generatePRBody(extraMsgs []string) string {
 	var assignment string
 	if err == nil {
 		if oncaller != "" {
-			assignment = "/assign @" + oncaller
+			assignment = fmt.Sprintf("/assign @%s\n/cc @%s\n", oncaller, oncaller)
```
Comment (chaodaiG) [inline tools/prow-auto-bumper/pullrequest.go]: are you sure `\n` works?

Comment (coryrc) [reply tools/prow-auto-bumper/pullrequest.go]: It's used in body just above without problem.

**Comment by chizhg** [comment]
It seems what we stated in https://github.com/knative/test-infra/issues/1638#issue-555923905 and https://github.com/knative/test-infra/pull/1661#issue-371018250 are both wrong, Prow won't automatically cc or assign approvers..
So after this change, the oncall will need to manually assign one approver if they don't have approve power, is it a reasonable flow for you? Or we can consider adding all oncalls to be OWNER of Prow configs?

**Comment by coryrc** [comment]
> So after this change, the oncall will need to manually assign one approver if they don't have approve power, is it a reasonable flow for you?

Yes. Today, #1737 made reviewers of yt3liu and steuhs, neither of whom can approve. So it was just bothering them.

> Or we can consider adding all oncalls to be OWNER of Prow configs?

This is good as well, but not up to me :).

**Pull request merged**